calico_rkt/calico_rkt.py

```diff
@@ -14,387 +14,473 @@
 
 from __future__ import print_function
 import socket
-import functools
 import logging
 import json
 import os
 import sys
 
-from subprocess import check_output, CalledProcessError, check_call
-from netaddr import IPAddress, IPNetwork, AddrFormatError
+from subprocess32 import CalledProcessError,  Popen, PIPE
+from netaddr import IPAddress, AddrFormatError
 
 from pycalico import netns
-from pycalico.ipam import IPAMClient, SequentialAssignment
-from pycalico.netns import Namespace
-from pycalico.datastore_datatypes import Rules, IPPool
-from pycalico.datastore import IF_PREFIX
-from pycalico.datastore_errors import PoolNotFound
+from pycalico.netns import Namespace, remove_veth
+from pycalico.datastore import DatastoreClient
+from pycalico.datastore_errors import MultipleEndpointsMatch
 
+# Calico Configuration Constants
 ETCD_AUTHORITY_ENV = 'ETCD_AUTHORITY'
-LOG_DIR = '/var/log/calico/calico-rkt'
 
 ORCHESTRATOR_ID = "rkt"
```

Comment:
I need to look into what libcalico uses this for, but I suspect we're going to want this to be dynamic (this plugin could be rkt, k8s, or something else).